### PR TITLE
Emit stress test metrics in Fortio format.

### PR DESCRIPTION
**Pull request by jkowalski**
This addresses the instrumentation aspect of #573 which is emitting performance metrics from E2E tests and saving them to a folder in JSON format compatible with Fortio.

By default stress test results are saved in `build/.perf` directory.

### CODE REVIEW

**Comment by markmandel** [comment]
Is this on hold until we're happy with #573 ?

**Comment by jkowalski** [comment]
> Is this on hold until we're happy with #573 ?

This is a small component of the solution, I'm hoping not a controversial one, unless we don't want to do Fortio at all.

File: test/e2e/framework/perf.go [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
+	"net/http"
+	"os"
+	"path/filepath"
 	"sync"
 	"time"
 
+	"fortio.org/fortio/fhttp"
+	"fortio.org/fortio/stats"
 	"github.com/sirupsen/logrus"
+
```
Comment (markmandel) [inline test/e2e/framework/perf.go]: nit: blank line.

File: test/e2e/framework/framework.go [hunk 2]
```diff
@@ -49,10 +49,11 @@ type Framework struct {
 	GameServerImage string
 	PullSecret      string
 	StressTestLevel int
+	PerfOutputDir   string
 }
 
 // New setups a testing framework using a kubeconfig path and the game server image to use for testing.
-func New(kubeconfig, gsimage string, pullSecret string, stressTestLevel int) (*Framework, error) {
+func New(kubeconfig string) (*Framework, error) {
```
Comment (markmandel) [inline test/e2e/framework/framework.go]: Curious reason for this - getting too wide?

**Comment by aLekSer** [comment]
@jkowalski Possibly we need to add a small description of `make stress-test-e2e` in a build/README.md file:
https://github.com/GoogleCloudPlatform/agones/blob/4ee2602c77bad8293d5dc8a86486ddd5c74fd844/build/README.md#L41
It is hard to guess the suitable cluster size needed to run test with `STRESS_TEST_LEVEL=20`
Currently only stress test levels <= 2 can be executed on default gcloud-test-cluster.

**Pull request merged**